### PR TITLE
Bulbs video carousel

### PR DESCRIPTION
**Pull request by collin**

### CODE REVIEW

**Comment by collin** [comment]
http://onion.local/special/lawn-and-garden/video/3943

@shawncook @spra85 @kand

File: bulbs/special_coverage/templates/special_coverage/bulbs_sc_landing.html [hunk 1]
```diff
@@ -1,121 +1,133 @@
...
+            name-only
+          >
+          </campaign-display>
+        {% endif %}
+        <bulbs-video-carousel-previous>&lt;</bulbs-video-carousel-previous>
+        <bulbs-video-carousel-next>&gt;</bulbs-video-carousel-next>
+      </div>
+
+      <bulbs-video-carousel-slider class='sc-video-grid'>
+        {% for char in 'abcdef' %}
```
Comment (collin) [inline bulbs/special_coverage/templates/special_coverage/bulbs_sc_landing.html]: Once the test link on onion is approved I'll take this line out.

File: bulbs/styles/static/styles/bulbs-styles/mixins/flex-grid.scss [hunk 2]
```diff
@@ -34,3 +35,19 @@
   $flexgrid_calc_gutter: $gutter * 1px;
   width: calc(#{$flexgrid_calc_percent} - #{$flexgrid_calc_gutter});
 };
+
+@mixin overflow-flexgrid($columns: 0, $gutter: 24) {
```
Comment (collin) [inline bulbs/styles/static/styles/bulbs-styles/mixins/flex-grid.scss]: This mixin is similar to the flexgrid, but it doesn't wrap the items.

**Comment by MelizzaP** [comment]
cool_thumb_by_songbird79

**Pull request merged**